pages.fr/common/atuin.md

```diff
@@ -0,0 +1,29 @@
+# atuin
+
+> Enregistre votre historique shell dans une base de donnée.
+> Peut également synchroniser votre historique chiffré entre plusieurs machines.
+> Plus d'informations : <https://atuin.sh/docs/overview/introduction/>.
+
+- Installe atuin dans votre shell :
+
+`eval "$(atuin init {{bash|zsh|fish}})"`
+
+- Importe l'historique du shell par défaut :
+
+`atuin import auto`
+
+- Recherche dans l'historique shell une commande spécifique :
+
+`atuin search {{commande}}`
+
+- Inscrire un compte sur le serveur de synchronisation par défaut :
+
+`atuin register -u {{nom_d_utilisateur}} -e {{email}} -p {{mot_de_passe}}`
+
+- Connexion au serveur de synchronisation par défaut :
```

Comment:
Hey, I checked this command description using Google Translate it read `Connection to the default synchronization server:`. I have a suggestion
```suggestion
- Connectez-vous au serveur de synchronisation par défaut :
```
Is this better, or is the current one correct?